src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -304,6 +304,21 @@ public ResponseEntity<byte[]> getCourseIcon(@PathVariable Long courseId) {
         return responseEntityForFilePath(fileService.actualPathForPublicPath(course.getCourseIcon()));
     }
 
+    /**
+     * GET /files/templates/code-of-conduct : Get the Code of Conduct template
+     *
+     * @return The requested file, 403 if the logged-in user is not allowed to access it, or 404 if the file doesn't exist
+     */
+    @GetMapping("files/templates/code-of-conduct")
+    @EnforceAtLeastInstructor
+    public ResponseEntity<byte[]> getCourseCodeOfConduct() throws IOException {
+        var templatePath = Path.of("templates", "codeofconduct", "README.md");
+        log.debug("REST request to get template : {}", templatePath);
+        var resource = resourceLoaderService.getResource(templatePath);
+        var path = resource.getFile().getPath();
+        return responseEntityForFilePath(path);
```

Comment:
Please use `resource.getInputStream()` here to load the content, this won't work if the resource is bundled into the application.

Reply:
Thanks 0f80c151ebe75fdc2c3ab02640561b8977893adf